presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -224,6 +225,15 @@ public synchronized void setDatabaseOwner(HiveIdentity identity, String database
         writeSchemaFile("database", databaseMetadataDirectory, databaseCodec, new DatabaseMetadata(newDatabase), true);
     }
 
+    @Override
+    public Optional<String> getConfigValue(String name)
```

Comment:
Do you still need it?

Reply:
I thought it might not hurt to have it . Happy to remove it.

Reply:
I'll remove it.